themes/cardinal_service/cardinal_service.info.yml

```diff
@@ -30,3 +30,10 @@ libraries:
 
 ckeditor_stylesheets:
   - dist/css/ckeditor.css
+
+libraries-override:
+  # Replace an asset with another.
+  pdb_react/react:
+    js:
+      //cdnjs.cloudflare.com/ajax/libs/react/15.0.1/react.min.js: //unpkg.com/react@16/umd/react.production.min.js
+      //cdnjs.cloudflare.com/ajax/libs/react/15.0.1/react-dom.min.js: //unpkg.com/react-dom@16/umd/react-dom.production.min.js
```

Comment:
isn't `//` an anti-pattern now? Shouldn't you use `https://` if it is available on the remote resource?

Reply:
yeah i'm pretty sure you're right